### PR TITLE
chore(android/app,ios/app): Update crowdin strings for De

### PR DESCRIPTION
**Pull request by darcywong00**

### CODE REVIEW

**Review by mcdurdin — CHANGES_REQUESTED** [review]
Can you make sure the Localizable.strings files are converted to UTF-8 (e.g. using VS Code, or iconv) so we can do comparisons?

They do work as UTF-8.

**Comment by jahorton** [comment]
Might should've based this upon https://github.com/keymanapp/keyman/pull/4650, to make use of its changes in those regards.

**Comment by darcywong00** [comment]
I ran `iconv -f utf-16 -t utf-8 Localizable.strings` but afict nothing changed in Git.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# iOS localizable strings
```
Comment (mcdurdin) [inline .gitattributes:1]: I think we should avoid this if we can.

Can we instead just convert the files (by hand at first) to utf-8 before adding them to the repo?

Then in the future, we can add that into the download script if necessary.

**Comment by darcywong00** [comment]
Abandoning this for #4672

**Pull request closed without merging**